res/controllers/American-Audio-VMS4-scripts.js

```diff
@@ -392,11 +398,40 @@ VMS4.jog_move_msb = function(channel, control, value, status, group) {
    deck.jogMsb = value;
 }
 
-VMS4.touch_strip = function(channel, control, value, status, group) {
-   // Only modify the playposition if the deck is NOT playing!
-   if (engine.getValue(group, "play") === 0) {
-       engine.setValue(group, "playposition", value/0x7F);
-   }
+// VMS4.touch_strip = function(channel, control, value, status, group) {
+//    // Only modify the playposition if the deck is NOT playing!
+//    if (engine.getValue(group, "play") === 0) {
+//        engine.setValue(group, "playposition", value/0x7F);
+//    }
+// }
+
+VMS4.strip_touch = function(channel, control, value, status, group) {
+    var deck = VMS4.GetDeckNum(group);
+    if (deck === 1) {    // Left side
+        if ((status & 0xF0) === 0x90) {    // If button down
+            // TODO: Need a CO to focus explicitly on category selection pane
+            //  "[Library]","MoveFocus" just toggles and is frustrating
+        } else {    // Button up
+            VMS4.touchStripPos["Left"] = null;
+        }
+    } else {    // Right side
+        if ((status & 0xF0) === 0x90) {    // If button down
+            // TODO: Need a CO to focus explicitly on track selection pane
+            //  "[Library]","MoveFocus" just toggles and is frustrating
+        } else {    // Button up
+            VMS4.touchStripPos["Right"] = null;
+        }
+    }
+}
+
+VMS4.strip_scroll = function(channel, control, value, status, group) {
+    var side = VMS4.StripToSide[control];
+    if (VMS4.touchStripPos[side] != null) {
```

Comment:
`!==`

Reply:
I'll fix that. (What's the difference again?)

Reply:
`!=` can easily introduce confusing bugs because of JS' bizarre type coercion rules.